### ISSUE TITLE
What are the differences between "cls_outputs" and "pred_class_logits" in embedding_head?

### ISSUE THREAD

**Issue by ShowTsai** [JDAI-CV/fast-reid#318]
As the title, what are the differences between them?
Are they the same when the classifier is ```linear```?

**Comment by L1aoXingyu** [comment]
There is no difference when using softmax classifier, but they are different when using margin-based softmax, such as arcface and circle softmax.

**Comment by ShowTsai** [comment]
So what are the exact differences between them?
Thanks.

**Comment by ShowTsai** [comment]
There is a suggestion:
replace the  function ```cross_entropy_loss``` parameter name ```pred_class_logits``` to ```cls_outputs```

**Comment by L1aoXingyu** [comment]
I will take your suggestion.
